src/odinapi/templates/base.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <meta name="keywords" content="footer, address, phone, icons" />
     <meta name="referrer" content="no-referrer">
+    <link rel="icon" type="image/png" href="{{ url_for('static', filename='images/favicon.ico') }}">
```

Comment:
is the mime really image/png ?

Reply:
no, but it appears to work anyway...

> $ identify odinapi/static/images/favicon.ico 
> odinapi/static/images/favicon.ico ICO 16x16 16x16+0+0 24-bit sRGB 894B 0.000u 0:00.000

Changed to `x-icon`.